python/python/lancedb/table.py

```diff
@@ -1851,7 +1851,7 @@ def create(
             data but will validate against any schema that's specified.
         on_bad_vectors: str, default "error"
             What to do if any of the vectors are not the same size or contains NaNs.
-            One of "error", "drop", "fill".
+            One of "error", "drop", "fill", "null".
```

Comment:
i thought fill is full with Nans?

Reply:
Fill is fill vectors with values. For example, the input (two 2-dimensional vectors):

```
[[2.0, NaN], [1.0, 3.0]]
```
Becomes (with default fill value 0.0):

```
[[2.0, 0.0], [1.0, 3.0]]
```

`null` replaces the entire vector with null, so that same input would become:

```
[null, [1.0, 3.0]]
```

Reply:
Got it, could you help add that info in the doc?
kind of confusing on what the fill fills

Reply:
Added a section in the user guide.